addons/danger/functions/fnc_tacticsAssess.sqf

```diff
@@ -102,7 +103,7 @@ if !(_enemies isEqualTo [] || {_unitCount < random 3}) then {
     // enemies within X meters of leader and either attacker or unit is inside buildings
     private _inside = _unit call EFUNC(main,isIndoor);
     private _nearIndoorTarget = _enemies findIf {
-        _unit distance2D _x < (GVAR(cqbRange) * 1.4)
+        _unit distance2D _x < RANGE_NEAR
```

Comment:
Are we intentionally removing configurability here?

Reply:
Yes. 
I feel that layers and layers of hidden interaction with the CQB Range setting is probably less clear than it should be. I haven't checked the profile changes yet, but I suspect that sort of configuration for _group tactics_ is better handled there.